tests/console/sshd.pm

```diff
@@ -73,6 +73,7 @@ sub run {
 sub test_cryptographic_policies() {
     my %args = @_;
     my $remote_user = $args{remote_user};
+    my $system_role = $args{system_role};
```

Comment:
you don't need this anymore and I guess you can remove WIP.

Reply:
updated.